README.md

```diff
@@ -31,7 +31,7 @@ Use following Makefile commands:
 # Testing
 To setup accounts passphrase you need to setup an environment variable: `export ACCOUNT_PASSWORD="secret_pass_phrase"`.
 
-To test statusgo, use: `make ci`.
+To test statusgo, use: `make ci`. Make sure the `gometalinter` package is installed: `go get -u github.com/alecthomas/gometalinter && go install github.com/alecthomas/gometalinter && export PATH=$PATH:$GOPATH/bin`
```

Comment:
There is [`make lint-install` target](https://github.com/status-im/status-go/blob/develop/static/tools/mk/lint.mk) available. So you can change it to:

> Make sure to run `make lint-install` if you run tests for the first time.